pubsub.go

```diff
@@ -570,13 +605,29 @@ func (p *PubSub) handleIncomingRPC(rpc *RPC) {
 				p.topics[t] = tmap
 			}
 
-			tmap[rpc.from] = struct{}{}
+			if _, ok = tmap[rpc.from]; !ok {
+				tmap[rpc.from] = struct{}{}
+				if subs, ok := p.myTopics[t]; ok {
+					inboundPeer := rpc.from
```

Comment:
call this just `peer`.

Reply:
sure, didn't want to obscure the `peer` pkg, but that's fine